src/xlsx/functions/charts.ts

```diff
@@ -211,6 +211,63 @@ function insertTextProperties(
   `;
 }
 
+function extractTrendline(trend: ExcelChartDataset["trend"]) {
+  if (!trend) {
+    return "";
+  }
+  if (trend.type === "poly" && trend.order) {
+    if (trend.order > 1) {
+      return escapeXml/*xml*/ `
+        <c:trendline>
+            ${extractTrendlineCommonAttributes(trend)}
+            <c:trendlineType val="poly" />
+            <c:order val="${trend.order}" />
+        </c:trendline>
+      `;
+    }
+    return escapeXml/*xml*/ `
+    <c:trendline>
+        ${extractTrendlineCommonAttributes(trend)}
+        <c:trendlineType val="linear" />
+    </c:trendline>
+  `;
+  }
+  if (trend.type === "movingAvg") {
+    return escapeXml/*xml*/ `
+      <c:trendline>
+          ${extractTrendlineCommonAttributes(trend)}
+          <c:trendlineType val="movingAvg" />
+          <c:period val="${trend.window}" />
+      </c:trendline>
+    `;
+  }
+  return escapeXml/*xml*/ `
+    <c:trendline>
+        ${extractTrendlineCommonAttributes(trend)}
+        <c:trendlineType val="${trend.type}" />
+    </c:trendline>
+  `;
+}
+
+function extractTrendlineCommonAttributes(trend: ExcelChartDataset["trend"]) {
+  if (!trend) {
+    return "";
+  }
+  return escapeXml/*xml*/ `
+    <c:spPr>
+      <a:ln w="19050" cap="rnd">
+          <a:solidFill>
+              <a:srgbClr val="${trend.color ? toXlsxHexColor(trend.color) : "000000"}" />
```

Comment:
if there is no color defined in the definition, we should probably apply the same logic we do for the runtime drawing and not put "black". Or maybe we don't have to specify the key.